packages/design-tokens/src/js/typography/types.ts

```diff
@@ -25,7 +24,6 @@ export type ThemeTypography = {
   lDisplayMD: TypeStyle;
   lHeadingLG: TypeStyle;
   lHeadingMD: TypeStyle;
-  lHeadingSMRegular: TypeStyle;
   lHeadingSM: TypeStyle;
```

Comment:
Removing deprecated tokens from types. Ensured no instances left in codebase

<img width="300" alt="Screenshot 2025-05-25 at 2 33 22 PM" src="https://github.com/user-attachments/assets/b8d2fe71-cb8c-4949-a8e6-5d7d5c1a19c7" /><img width="300" alt="Screenshot 2025-05-25 at 2 34 04 PM" src="https://github.com/user-attachments/assets/bacd6c62-e22e-458f-8c4c-9586a1738491" />